### PR TITLE
Add country calling codes

### PR DESCRIPTION
**Pull request by Samuell1**

### CODE REVIEW

**Comment by bennothommo** [comment]
@Samuell1 instead of calling code, could it be called "phone prefix" - I feel that's more descriptive of what it is. The field should be `phone_prefix`.

**Comment by LukeTowers** [comment]
@bennothommo I believe the official name is "International Calling Code": https://en.wikipedia.org/wiki/List_of_mobile_telephone_prefixes_by_country, so calling_code seems fine to me.

**Comment by bennothommo** [comment]
I stand corrected :)

**Comment by Samuell1** [comment]
@LukeTowers @bennothommo yep i did same naming as wiki :D and other packages use same too

File: updates/seed_callingcodes.php [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            'VN' => array('name' => 'VIET NAM', 'code' => '84'),
+            'VU' => array('name' => 'VANUATU', 'code' => '678'),
+            'WF' => array('name' => 'WALLIS AND FUTUNA', 'code' => '681'),
+            'WS' => array('name' => 'SAMOA', 'code' => '685'),
+            'XK' => array('name' => 'KOSOVO', 'code' => '381'),
+            'YE' => array('name' => 'YEMEN', 'code' => '967'),
+            'YT' => array('name' => 'MAYOTTE', 'code' => '262'),
+            'ZA' => array('name' => 'SOUTH AFRICA', 'code' => '27'),
+            'ZM' => array('name' => 'ZAMBIA', 'code' => '260'),
+            'ZW' => array('name' => 'ZIMBABWE', 'code' => '263')
```
Comment (LukeTowers) [inline updates/seed_callingcodes.php:241]: We don't use array(), we just use the short form

File: updates/seed_callingcodes.php [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        );
+
+        foreach ($callingCodes as $key => $item) {
+            Country::where('code', $key)->update(['callingcode' => $item['code'] ]);
+        }
+
+
```
Comment (LukeTowers) [inline updates/seed_callingcodes.php:248]:
```suggestion
```

File: updates/seed_callingcodes.php [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    }
```
Comment (LukeTowers) [inline updates/seed_callingcodes.php:249]:
```suggestion
        }
    }
```

File: updates/seed_callingcodes.php [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+            'VN' => ['name' => 'VIET NAM', 'code' => '84'],
+            'VU' => ['name' => 'VANUATU', 'code' => '678'],
+            'WF' => ['name' => 'WALLIS AND FUTUNA', 'code' => '681'],
+            'WS' => ['name' => 'SAMOA', 'code' => '685'],
+            'XK' => ['name' => 'KOSOVO', 'code' => '381'],
+            'YE' => ['name' => 'YEMEN', 'code' => '967'],
+            'YT' => ['name' => 'MAYOTTE', 'code' => '262'],
+            'ZA' => ['name' => 'SOUTH AFRICA', 'code' => '27'],
+            'ZM' => ['name' => 'ZAMBIA', 'code' => '260'],
+            'ZW' => ['name' => 'ZIMBABWE', 'code' => '263']
```
Comment (LukeTowers) [inline updates/seed_callingcodes.php:240]: Why are the name values required? Also could you add an inline comment referencing where you got the values from? Finally, I think the column in the DB should be `calling_code`, not `callingcode`

File: updates/seed_callingcodes.php [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        ];
+
+        foreach ($callingCodes as $key => $item) {
+            Country::where('code', $key)->update(['callingcode' => $item['code'] ]);
+        }
```
[the remark this answers is not in the document]
Comment (Samuell1) [reply updates/seed_callingcodes.php:245]: Not sure how i do that.

File: updates/seed_callingcodes.php (hunk 4 above)
Comment (Samuell1) [reply updates/seed_callingcodes.php:240]: I changed it to calling_code, i used to export csv to php array then i used names to return to my code then i reused them here, if you want i can remove them later.

File: lang/en/lang.php [hunk 6]
```diff
@@ -66,5 +66,7 @@
         'code_comment' => 'Enter a unique code to identify this country.',
         'enabled' => 'Enabled',
         'pinned' => 'Pinned',
+        'callingcode' => 'Calling code',
+        'callingcode_comment' => 'Numerical country calling code without the plus sign.',
```
Comment (LukeTowers) [inline lang/en/lang.php:70]: should be calling_code here too

File: updates/seed_callingcodes.php (hunk 5 above)
[the remark this answers is not in the document]
Comment (LukeTowers) [reply updates/seed_callingcodes.php:245]: Still need a reference as to where you got the info from

[the remark this answers is not in the document]
Comment (Samuell1) [reply updates/seed_callingcodes.php:245]: I dont know, it was some random list somewhere

[the remark this answers is not in the document]
Comment (Samuell1) [reply updates/seed_callingcodes.php:245]: Maybe it was this list? https://gist.github.com/josephilipraja/8341837

**Comment by daftspunk** [comment]
Thanks @Samuell1!

**Pull request merged**